cpp/src/arrow/builder.cc

```diff
@@ -25,6 +25,25 @@
 
 namespace arrow {
 
+Status ArrayBuilder::AppendToBitmap(bool is_null) {
```

Comment:
Makes sense, I did it this way because it was closer to how the code previously worked.  But I will change it.